docs/open-source-tizen/porting/system.md

```diff
@@ -2095,130 +2023,122 @@ sensorhub_device::parse_debug(const char *hub_data, int data_len)
-, m_data_node(-1)
-{
-}
+, m_data_node(-1) {}
 
-sensorhub_controller::~sensorhub_controller()
-{
-}
+sensorhub_controller::~sensorhub_controller() {}
 
-sensorhub_controller& sensorhub_controller::get_instance(void)
-{
+sensorhub_controller& sensorhub_controller::get_instance(void) {
     static sensorhub_controller instance;
 
     return instance;
 }
 
 int
-sensorhub_controller::get_poll_fd(void)
-{
+sensorhub_controller::get_poll_fd(void) {
     /* Returns the sensorhub fd */
 
     return -1;
 }
 
 bool
-sensorhub_controller::enable(void)
-{
+sensorhub_controller::enable(void) {
     m_enabled = true;
     INFO("Enable Sensorhub");
 
     return true;
 }
 
 bool
-sensorhub_controller::disable(void)
-{
+sensorhub_controller::disable(void) {
     m_enabled = false;
     INFO("Disable Sensorhub");
 
     return true;
 }
 
 int
-sensorhub_controller::open_input_node(const char* input_node)
-{
+sensorhub_controller::open_input_node(const char* input_node) {
     /* Implements the specific sensorhub logic */
 
     return -1;
 }
 
 bool
-sensorhub_controller::read_fd(sensorhub_data_t &data)
-{
+sensorhub_controller::read_fd(sensorhub_data_t &data) {
     /* Implements the specific sensorhub logic */
 
     return false;
 }
 
 int
-sensorhub_controller::read_sensorhub_data(void)
-{
+sensorhub_controller::read_sensorhub_data(void) {
     /* Implements the specific sensorhub logic */
 
     return -1;
 }
 
 int
-sensorhub_controller::read_large_sensorhub_data(void)
-{
+sensorhub_controller::read_large_sensorhub_data(void) {
     /* Implements the specific sensorhub logic */
 
     return -1;
 }
 
 int
-sensorhub_controller::send_sensorhub_data(const char *data, int data_len)
-{
+sensorhub_controller::send_sensorhub_data(const char *data, int data_len) {
     /* Implements the specific sensorhub logic */
 
     return -1;
 }
 
 int
-sensorhub_controller::print_sensorhub_data(const char* name, const char *data, int length)
-{
+sensorhub_controller::print_sensorhub_data(const char* name, const char *data, int length) {
     /* Implements the specific sensorhub logic */
 
     return 0;
 }
-
 ```
 
 ### References
 
-The reference kernel configuration for sensors varies with different vendor types.
-However, there is a standard kernel subsystem for sensors, that is IIO(The Industrial I/O subsystem) subsystem. IIO is intended to provide support for devices that in some sense are analog to digital or digital to analog.
+The reference kernel configuration for sensors varies depending on vendor types. However, there is a standard kernel subsystem for sensors: the Industrial I/O (IIO) subsystem. IIO is intended to provide support for devices that are, in some sense, analog to digital or digital to analog.
 For more information, see [https://wiki.analog.com/software/linux/docs/iio/iio](https://wiki.analog.com/software/linux/docs/iio/iio).
 
-Examples)
+The following table lists some example kernel configurations.
+
+**Table: Example kernel configurations**
```

Comment:
Should the same device nodes be listed for all table rows, or is it correct to have no content for 3 rows?

Reply:
Keep the current base commit.

Reply:
OK